synfig-core/src/synfig/value.cpp

```diff
@@ -210,16 +218,15 @@ ValueBase::clear()
 		func(data);
 	}
 	ref_count.detach();
-	data=0;
+	data=nullptr;
 	type=&type_nil;
 }
 
-
 Type&
 ValueBase::ident_type(const String &str)
 {
 	Type *type = Type::try_get_type_by_name(str);
-	return type == NULL ? type_nil : *type;
+	return !type ? type_nil : *type;
```

Comment:
Maybe better to invert the condition now? Like:
```
return type ? *type : type_nil;
```

Reply:
fixed and squashed